test/fft_utest.cpp

```diff
@@ -140,6 +144,68 @@ class FftTest : public ::testing::Test {
             ASSERT_EQ(ifft_1, v);
         }
     }
+
+    /**
```

Comment:
Squash the two lines, should be `/** Convert a number into a vector of digits padded with zeros.`